lib/stats-collector.js

```diff
@@ -15,14 +22,16 @@
  */
 
 /**
- * Provides stats such as test duration,
- * number of tests passed / failed etc.
+ * Provides stats such as test duration, number of tests passed / failed etc., by listening for events emitted by `runner`.
  *
- * @public
- * @memberof Mocha
- * @param {Runner} runner
+ * @private
```

Comment:
I have attempted to ask in multiple places now. Why are you changing everything to `@private`? Attempting to make this mean "user no touchy" is **not** what the tag is for -- it's meant to document the _module_, not the _user_ access.

For example, it's considered incorrect for _anything_ exported from a module to be private [Sorry, lost my reference], and there was an issue to make this official.

Closure Compiler documents the module POV as well. [If tagged `@private`, it's **unavailable** outside _this_ file](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#private).

Reply:
assigning @plroebuck for response

Reply:
though I suppose it's actually closed.

Reply:
Perhaps we should think of both from now on? (E.g stats-collector @public and @ignore).
It's usful to have code documented even if it doesn't show in generated source.

Reply:
@craigtaub if we do that then it's fair game as a public API, even if it's not documented on the site as such.

What I like about the `@private` solution is that you can generate "complete" API docs with a flag.

regardless of any other solutions we decide on, I feel `@ignore` is appropriate for functions that aren't exported.

Reply:
Fair point. I don't really mind what we decide as long as we can be consistent.